packages/amplify-graphql-api-construct-tests/src/__tests__/restricted-field-auth/graphql-schemas/gen2/schema-primary.graphql

```diff
@@ -0,0 +1,7 @@
+type Primary @model @auth(rules: [{ allow: public, operations: [read] }, { allow: owner }]) {
```

Comment:
Broke the gen2 schema into 2 files so we can mix & match primary & related between our data sources